sim-lib/src/lib.rs

```diff
@@ -35,15 +40,37 @@ pub struct ActivityDefinition {
     pub amt_msat: u64,
 }
 
-// Phase 2: Event Queue
+#[derive(Debug, Error)]
+pub enum PaymentError {
```

Comment:
I think we should just change this to `Error`

Reply:
fleshed out error def more so we can report ln errors better